packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -39,6 +39,8 @@ function webpackConfigSingleSpa(opts) {
 
   let HtmlWebpackPlugin = opts.HtmlWebpackPlugin || _HtmlWebpackPlugin;
 
+  const outputSystemJS = !!opts.outputSystemJS;
```

Comment:
Force a `true` or `false` rather than allowing undefined/null to be handled differently.